tutorials/examples/preprocess_imgs.py

```diff
@@ -0,0 +1,92 @@
+"""
+This script illustrates how to preprocess experimental CryoEM images
+before performing reconstructing 3D map.
+"""
+
+import logging
+import matplotlib.pyplot as plt
+
+from aspire.source.relion import RelionSource
+from aspire.estimation.noise import WhiteNoiseEstimator
+
+logger = logging.getLogger('aspire')
+
+starfile_in = '/tigress/junchaox/CryoEMdata/empiar10028/shiny_2sets.star'
+data_folder = '/tigress/junchaox/CryoEMdata/empiar10028/'
+pixel_size = 1.34
+num_imgs = 1024
+
+logger.info('This script illustrates how to preprocess experimental CryoEM images')
+logger.info(f'Read in images from {starfile_in} and preprocess the images')
+source = RelionSource(
+    starfile_in,
+    data_folder,
+    pixel_size=pixel_size,
+    max_rows=num_imgs
+)
+
+logger.info('Obtain original images')
+imgs_od = source.images(start=0, num=num_imgs).asnumpy()
```

Comment:
Why get all the images if we only need to display the first one?

Reply:
Made changes to return only a few of them.

Reply:
Ok so now we're getting ten but only using one. Why not just get that single image?

Reply:
Some preprocess steps requite calculate averages. I thing It will be more reasonable using some of them instead one. Changed to one image in new version.